Kenzo

kenzo251 opened the pull request:
i'll do the code examples tomorrow & if i can find it the integration

kenzo251 commented:
i accidentally had app.js in lowercase..... though i think it is possible to fix on your end

srfoster commented:
This is awesome!!!! :)

We can move toward accepting this if:

* It doesn't override the routing conventions for "/docs/:pagename" (i.e. don't add a route in App.js).
* Instead, alter the code that already renders out the docs at "/docs/langs", such that it renders the new documentation.
* For the drawer (within a drawer), we'll have to see how that looks.  Might have to make some changes there (or maybe not).
* I'm also on the fence about whether to keep the data in the rawdata.json file, or to move it to the Racket lang server (whose job is to keep track of the various magical languages that can execute in orb world).  But we can cross that bridge in a future iteration.

The UI looks great, though!  I'm excited to get this in!

kenzo251 commented:
so uh, i have a new commit on my fork(and branch on there) however there seems to be no update on this pull request
added files /CodespellsDocs/card.js; drawer.js; Index.js; rawData.js and updated Docs/Docs.js to display the new docs, however am having some problems with margins for the underlying paper or card on that docs page regarding the drawer(which is now on the right) since i don't know where the margins for this element are located
